### ISSUE TITLE
IT Support: Unhiding the `Signs | New Work Order` button for MMC Work Orders

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#14703]
Allyson called because she was trying to create a Signs Work Order from an MMC ticket she created, but was unable to see these buttons: 
![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/ff19ed95-b32a-4278-9d8c-a6a7da2eb319)
 
Looking into the rules, I found this and was pretty sure this was causing that view to hide:
![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/144e32c9-c0d0-4087-8e20-cf2fe7ca439a)

- Kept the rule in place, but unhid the view from the page based on the rule. Allyson refreshed and was able to see the buttons. 
- Creating this issue because we still might want to hide the `View Flex Notes` button based on this rule, but don't want to hide the `Sign Work Orders` button.

**Comment by ChristinaTremel** [comment]
After changing Rule 1, Allyson let me know that she was unable to see the `Add mmc_activity` and `Add work_order`. 

I compared the page rules to the page rules in Test copy of data tracker from 05-11-23. Found that all the rules were matching. 

When investigating Prod & Test, these two rules looked backwards. Confirmed with Karo and Diana who also thought those looked backwards. I updated the rules from **ASSET ATTACHED** is `YES` to `NO` and immediately the forms were visible as expected.

   ![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/737627dc-61b3-40ac-b178-ba85dbd77015)
   ![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/98a59e58-d906-47d3-8760-be93174472f1)

Checked in with Allyson and she confirmed that everything looks correct.
